### ISSUE TITLE
Usable for mouse datasets?

### ISSUE THREAD

**Issue by bhavyaac** [BlishLab/scriabin#1]
Hello! This is a very exciting project, thank you for your hard work. 
I wanted to ask you here since I wasn't able to find this information in the manuscript: is Scriabin usable with mouse scRNA-seq datasets?

**Comment by ajwilk** [comment]
Hi there! Good question--yes, Scriabin can be used with any ligand-receptor pair database, and functions that require a ligand-receptor pair database (eg. ```GenerateCCIM```) will take a custom list of ligand-receptor pairs by setting ```database = "custom"``` and passing character vectors of ligands and corresponding receptors to arguments ```ligands``` and ```recepts```, respectively. 

Ultimately, I have plans to automatically support standard mouse ligand-receptor databases in the package but haven't implemented this quite yet. In the meantime the approach I described above will work. I will also make sure this gets documented appropriately. 

Do be aware there are caveats with using non-human data in the ligand activity predictions run by NicheNet (see https://github.com/saeyslab/nichenetr/issues/104)

I'll leave this open for the time being and circle back once I've documented as described
